AirCasting/Settings/SettingsView.swift

```diff
@@ -10,23 +10,91 @@ import Foundation
 
 struct SettingsView: View {
     let logoutController: LogoutController
-
+    @State private var isToggle : Bool = false
+    @State private var showModal = false
     var body: some View {
```

Comment:
this is a huge body function over here, let's try to cut it into smaller pieces like in the example above and to make it more readable this way